python/pp_ser.py

```diff
@@ -1,10 +1,16 @@
 #!/usr/bin/env python
-#This file is released under terms of BSD license`
-#See LICENSE.txt for more information
+# This file is released under terms of BSD license`
+# See LICENSE.txt for more information
 
 from __future__ import print_function
-import linecache
 
+import filecmp
+import linecache
+import os
+import re
+import shutil
+import sys
+import tempfile
```

Comment:
Well it is a recommendation of PEP 8 to not have multiple module import on the same line. Then when can accommodate the PEP 8 style to what we want